src/containers/Review/ReviewWrapper.tsx

```diff
@@ -107,7 +107,11 @@ const ReviewWrapper: React.FC<ReviewWrapperProps> = ({ structure }) => {
             orgName={org?.name as string}
           />
           <div id="review-home-content">
-            {!isMobile && <ReviewProgress structure={structure} />}
+            {isMobile && structure.stages.length <= 2 ? (
+              <ReviewProgress structure={structure} />
+            ) : (
+              !isMobile && <ReviewProgress structure={structure} />
+            )}
```

Comment:
Sorry, I think I put you wrong. Let's just leave it how it was (hide on mobile) for now, and we can look at this a bit later. I think it needs a bit more before consideration it can work on mobile.

(I'm also thinking that it's generally going to be most useful when there are a lot of stages, so it's less useful for only 1-2 stage applications anyway)